MapboxNavigation/RouteMapViewController.swift

```diff
@@ -287,6 +288,32 @@ class RouteMapViewController: UIViewController {
             mapView.removeArrow()
         }
     }
+
+    func updateCameraAltitude(for routeProgress: RouteProgress) {
+        guard mapView.tracksUserCourse else { return } //only adjust when we are actively tracking user course
+        
+        let zoomOutAltitude = MapConstants.zoomedOutManeuverAltitude
+        let defaultAltitude = MapConstants.defaultAltitude
+        let isLongRoad = routeProgress.distanceRemaining >= MapConstants.longManeuverDistance
+        
+        
+        let currentStepIsMotorway = currentStep.isMotorway
+        let nextStepIsMotorway = upComingStep?.isMotorway ?? false
+        let isExiting = (currentStepIsMotorway && !nextStepIsMotorway) //are we exiting a motorway?
+        let notOnMotorway = (!currentStepIsMotorway && !nextStepIsMotorway) //are we not on a motorway?
```

Comment:
Are the parens necessary here and the line above?

Reply:
They are superfluous. Good catch, will fix.